### ISSUE TITLE
"Functional" PQs no longer working properly

### ISSUE THREAD

**Issue by kamikaiser** [WarEmu/WarBugs#7812]
There seems to be an issue which came up after the latest patch where PQs which were previously functional no longer function. I can't confirm this is for ALL PQs, but I have tried several which are on the functional list in both T1 and T2 on order and Destro. Here is basically what is happenning:

- Progress is no longer being tracked for kills. I can kill 100 of the right mobs, but the counter does not go up.
- If however I kill a correct mob, then run out of the range of the PQ area QUICKLY,a kill is counted for the progress
- This also works if I die or enter a SC. As long as there are mob corpses on the ground, and I leave the PQ area in any manner, then the counter has gone up the next time I nter that PQ
- If I do not leave out of the PQ before that mob respawns, then even if I run out no kill is counted

Basically, rather than tracking actual kills, the PQs are instead counting the number of corpses for those mobs when nobody is within the PQ area.

As a result, All of these PQs are stuck on the final kill because it doesnt count the kill itself, only the corpses, and this appears to not properly trigger the PQ advancing to the next stage.

The same progress issue was also visible in every PQ in Shadowlands as far as I could tell. Killed several required mobs, but no progress in the tracker. Along with that, the PQs which require interacting with an object also seem to not be working, because of a similar issue. You can click on the objects, get the influence, but no PQ progress in the tracker.

**Comment by kamikaiser** [comment]
Update: Found a trick to forcing it to advance. You need to find mobs close enough tot he edge of the PQ that you can pull them out of the PQ range without resetting them (enough that the PQ tracker goes away) and kill them there. Since youre out of the PQ, it counts the kill instantly, and advances to next stage.

**Comment by kamikaiser** [comment]
Update 2: This appears to only be with Stage 1 mobs. If you manage to force the PQ past stage 1, the remaining stages progress normally without having to force it by leaving the PQ.

**Comment by kamikaiser** [comment]
Update 3: I noticed something odd in doing some further testing of this stuff in a PQ that might be related to this issue as well. The one in this example is with the "Gloomridge Copse PQ" in Shadowlands. Stage 1 requires interacting with "Dirt Patch", and that seemed to work fine. Stage 2 though has 2 requirements:
1) Kill 16 Gloomridge Forestrager
2) Interact with 5 Dirt Patch

The thing is, if you interact with a Dirt Patch 5 times instead of counting towards the Dirt Patch requirement, it adds to the kill 16 Gloomridge Forestrager requirement and sets it to 5/16. However, if you leave the PQ area just like with the "kill" PQs, and go back intot he PQ the 5 points are now shifted down to the Dirt Patch requirement properly. Same thing happens if you do the 5 dirt patches, then kill a Gloomridge Forestrager, it suddenly changes it to 1/16 and 5/5 instead of the 5/16 kills and 0/5 dirt patch.

**Comment by Luth-RoR** [comment]
Update 3 is, high likely,  a different problem.

**Comment by Soulwhisper** [comment]
Already known :) I know what you talking about Kamikaiser.
 #6935

**Comment by Luth-RoR** [comment]
Ok, it seems the whole problem is the same as it was before the last patch.

>There seems to be an issue which came up after the latest patch where PQs which were previously functional no longer function.
